src/DataModel/ProductData.js

```diff
@@ -0,0 +1,17 @@
+const products = [
```

Comment:
@prabhatVinsol product name should be capitalized

Reply:
Done